service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -64,6 +69,14 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
+        List<Element<HearingBooking>> element = defaultIfNull(
+            caseData.getHearingDetails(), wrapElements(HearingBooking.builder().build()));
+
+        String hearingDate = getHearingDate(element);
```

Comment:
String nextHearingDate = getNextHearingDate(hearings)

Reply:
hmm I am not sure getNextHearing is right either. The method gets the first hearing in the collection

Reply:
I have called getFirstHearingStartDate